docker-compose.dev.yml

```diff
@@ -10,29 +10,60 @@ services:
       - .:/app
       - /app/node_modules
     ports:
-      - ${NESTJS_PORT}:${NESTJS_PORT}
+      - 80:80
       - 9229:9229
     command: npm run start:dev
     env_file:
-      - .env
+      - .env.ci
     networks:
       - main
     depends_on:
+      - dgraph
+      - zero
       - nats
-      - redis
+      - ratel
```

Comment:
Why the addition of the dgraph components? I think they are not needed anymore 😃